README.md

```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.com/HewlettPackard/py-spiffe.svg?branch=master)](https://travis-ci.com/HewlettPackard/py-spiffe)
 
 ## Overview
-Initial work in progress to create a python library for SPIFFE.
+Initial work in progress to create a Python library for SPIFFE.
```

Comment:
I would remove the "Initial work in progress.." I would make it: Python library for SPIFFE.